xml_converter/generators/cpp_templates/class_template.cpp

```diff
@@ -94,10 +99,16 @@ vector<string> {{cpp_class}}::as_xml() const {
     }
     {% endif %}
     {% else: %}
+    {% if (attribute_variable.attribute_name == "alpha")%}
```

Comment:
Don't branch on arbitrary documentation strings. This would mean that if someone wanted to change the documentation they might also need to change this template file too.

Reply:
Understood. Reverting this.